src/track/serato/markers.cpp

```diff
@@ -236,6 +236,10 @@ SeratoMarkersEntryPointer SeratoMarkersEntry::parseID3(const QByteArray& data) {
                     hasEndPosition,
                     endPosition,
                     color,
+                    100,
```

Comment:
This can be probably removed?

Reply:
fixed, there was another occurrence that was too much... solved Compiler happy